lib/logstash/outputs/dynatrace.rb

```diff
@@ -18,8 +18,13 @@
 require 'logstash/outputs/base'
 require 'logstash/json'
 
+MAX_RETRIES = 5
```

Comment:
This means that we wait for at most 2^4 seconds, and the whole tests in the worst case will take ~30 seconds, right? This seems reasonable to me.

Reply:
yes